tests/attacks/test_adversarial_patch.py

```diff
@@ -99,14 +115,17 @@ def test_pytorch(self):
         x_train = np.reshape(self.x_train_mnist, (self.n_train, 1, 28, 28)).astype(np.float32)
 
         attack_ap = AdversarialPatch(
-            ptc, rotation_max=22.5, scale_min=0.1, scale_max=1.0, learning_rate=5.0, batch_size=10, max_iter=500
+            ptc, rotation_max=0.5, scale_min=0.4, scale_max=0.41, learning_rate=5.0, batch_size=10, max_iter=500
         )
+        master_seed(seed=1234)
+        target = np.zeros(self.x_train_mnist.shape[0])
+        patch_adv, _ = attack_ap.generate(x_train, target)
 
-        patch_adv, _ = attack_ap.generate(x_train)
+        # patch_adv = np.reshape(patch_adv, (28, 28, 1)).astype(np.float32)
```

Comment:
If commenting out is desired, can we also remove the line?

Reply:
I agree